js/views/userPageVw.js

```diff
@@ -1413,6 +1437,92 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
+
+    $targ.addClass('loading');
+
+    var user = this.model.get('user');
+    var self = this;
+
+    var modList = {};
+    modList.moderators = user.moderators;
+    modList.moderators[modList.moderators.length] = this.userProfile.get('profile').guid;
+    user.moderators = modList.moderators;
+
+    saveToAPI('', user, user.serverUrl + "settings",
+      function(){
+        // confirmed
+        $targ.removeClass('loading');
+      },
+      function(){
```

Comment:
This is the failed function, if you put a function here it prevents the default error message from appearing. This should either be '', null, or add custom error handling inside the function. 

Look at how it's handled in settingsVw.js. The userPageVw.js is one of the oldest files in this project, it has a lot of examples of patterns we don't use any more.